### DISCUSSION TITLE
Kendo jQuery Build Numbers and kendo less files

### DISCUSSION THREAD

**Discussion by smchargue** [telerik/kendo-themes#3167] [Q&A] [answered]
I saw once a few months ago a page somewhere with a table that listed the @progress-kendo-bootstrap less file version and precisely what kendo jQuery build it goes with.  

That ring a bell with anyone?   I cannot find it anywhere now, and it was very valuable.

**Comment by cathrholm** [comment, marked as answer]
I haven't seen one for less, but came across this one for scss: https://docs.telerik.com/kendo-ui/styles-and-layout/sass-version-compatibility

**Reply by smchargue** [reply]
Thank you.  I actually meant SASS, not less.   I had seen that before, and then couldn't find it - man it was staring me right in my face

**Reply by cathrholm** [reply]
Haha, no worries. I find the documentation for kendo in general a bit confusing, i think it's difficult to find things, so i totally understand.

Btw, does the compatibility in that table make sense to you? 
I'm running "Kendo UI 2021.2.616 (R2 2021 SP1)", and I have version 4.43 for kendo-bootstrap-theme, but in the table it says version 4.35.1. I guess the table is meant to be read as "at least that version of themes" for  specific kendo-version? So that in my case I can use kendo-bootstrap-theme 4.35.1 or newer?

**Reply by smchargue** [reply]
I took that table to be more literal.  That why i was looking for it again. I think there could be some issues when you use an older version of the kendo library and grab the newest version of the sass build.  But I have no hard evidence on that. Just a feeling based on how disruptive the kendo build that changed listview html structure was for us.

**Reply by cathrholm** [reply]
Yeah, I guess I was just lucky.  I tried to upgrade kendo-theme-bootstrap from 4.43 to 4.44 now, and that was not compatible at all. It could be that I have the wrong version of bootstrap as well. That's another factor to take into account.
